virtool/users/data.py

```diff
@@ -219,9 +229,19 @@ async def update(self, user_id: str, data: UpdateUserRequest):
                 permissions,
             )
 
-        user = await fetch_complete_user(self._authorization_client, self._mongo, self._pg, user_id)
+        user = await fetch_complete_user(
+            self._authorization_client, self._mongo, self._pg, user_id
+        )
 
         if user is None:
             raise ResourceNotFoundError
 
         return user
+
+    async def check_is_empty(self) -> bool:
+        """
+        Checks if the "users" collection is empty.
+        """
+        if await self._mongo.users.count_documents({}) == 0:
+            return True
+        return False
```

Comment:
1. The query is not efficient... I know I wrote it in the issue. This is going to go through the whole collection to get a count even though we only want to know if the count is greater than `0`.

   Change to `await self._mongo.users.count_documents({}, limit=1) == 0`. This will stop the count as soon as one document is found.
2. The `return` can be cut to a single line.